health-services/project-factory/src/server/utils/sheetManageUtils.ts

```
@@ -0,0 +1,356 @@
+        else{
+            sheetMap[sheetName] = {
+                dynamicColumns: mergeAndGetDynamicColumns(null, sheet?.schema, localizationMap),
+                data: []
+            }
+        }
+    }
+}
+
+
+function mergeAndGetDynamicColumns(dynamicColumns: any, schema: any, localizationMap: any) : any {
+    if (!dynamicColumns) dynamicColumns = {};
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)